### PR TITLE
Boot fixes for Gen3

### PR DESCRIPTION
**Pull request by firscity**
Fixes for GIC code and image placing configuration for DomD U-boot

### CODE REVIEW

File: include/configs/rcar3-xen.h [hunk 1]
```diff
@@ -5,4 +5,11 @@
 
 #define CONFIG_LNX_KRNL_IMG_TEXT_OFFSET_BASE    CONFIG_SYS_LOAD_ADDR
 
+#undef CFG_EXTRA_ENV_SETTINGS
+#define CFG_EXTRA_ENV_SETTINGS		\
+	"fdt_high=0xffffffff\0"		\
+	"initrd_high=0xffffffff\0"
+
+//"bootm_low=0x70000000\0"	"bootm_size=0x20000000\0"
```
Comment (LKomaryanskiy) [inline include/configs/rcar3-xen.h]: Do we need this comment?

Comment (firscity) [reply include/configs/rcar3-xen.h]: thank you, fixed

**Review by lorc — APPROVED** [review]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by LKomaryanskiy** [comment]
`Reviewed-by: Leonid Komarianskyi <leonid_komarianskyi@epam.com>`

**Comment by firscity** [comment]
Added review tags, ready for merge. @lorc, can you please submit it?

**Pull request merged**